apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -0,0 +1,515 @@
+import ClipboardJS from 'clipboard';
+
+$(() => {
```

Comment:
I appreciate that this file contains very little jQuery. Should we eliminate it entirely? We'd need to copy over `src/lib/document-ready.ts` from PT and make a few other minor changes below.

Reply:
It still uses `tooltip`, `popover` and `modal` from bootstrap, which in v4 still rely on jQuery, so eliminating it entirely is not possible, unfortunately.

Reply:
Ah, ok, fair enough!